File: ts/iq/encrypt~get~key+user.ts
/** <iq type="get" xmlns="encrypt">. Replies with one of: Success, RequestError, ServerError. */
export function buildPreKeysFetchKeyBundles(contextJid?: string, jid: string): BinaryNode {
  return { tag: 'iq', attrs: { type: 'get', xmlns: 'encrypt', to: 's.whatsapp.net' }, content: [
      { tag: 'key', attrs: { pqsupport: 'true', context_jid: contextJid } },
    ] }
}

File: go/iq/encrypt~get~key+user.go
// BuildPreKeysFetchKeyBundles builds <iq type="get" xmlns="encrypt">. Replies with one of: Success, RequestError, ServerError.
func BuildPreKeysFetchKeyBundles(contextJid types.JID, jid types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"type": "get", "xmlns": "encrypt", "to": "s.whatsapp.net"},
		Content: []waBinary.Node{
			{Tag: "key", Attrs: waBinary.Attrs{"pqsupport": "true", "context_jid": contextJid}},
		},
	}
}